views/src/styles.css

```diff
@@ -0,0 +1,133 @@
+@import url("https://fonts.googleapis.com/css2?family=Quicksand&display=swap");
```

Comment:
This PR does not have UI yet, so please remove this file.

On the other hand, it looks to me that defining style here defeat the purpose of using tailwind.